compiler/rustc_codegen_ssa/src/target_features.rs

```diff
@@ -215,6 +215,7 @@ const HEXAGON_ALLOWED_FEATURES: &[(&str, Option<Symbol>)] = &[
 const POWERPC_ALLOWED_FEATURES: &[(&str, Option<Symbol>)] = &[
     // tidy-alphabetical-start
     ("altivec", Some(sym::powerpc_target_feature)),
+    ("power10-vector", Some(sym::powerpc_target_feature)),
```

Comment:
Can you move this below power10-vector, so they're in order?

Reply:
I think they're in alphabetical order, because `power10-vector` < `power8-altivec` (`1` < `8`) ?